angular/wolox-books/src/styles.scss

```diff
@@ -1 +1,8 @@
 /* You can add global styles to this file, and also import other style files */
+@import 'app/scss/equalizer';
+@import 'app/scss/fonts';
+
+body,
+html {
+  height: 100%;
```

Comment:
to achieve this we tend to say `min-height: 100vh` since 100% is relative to a parent's size and the html has no parent.